.changeset/long-spies-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/sidebar-settings": patch
```

Comment:
Not sure if its only a patch as its a breaking change, also not sure if we can change it that easily as if someone is using it on a Block (external devs), the existing block will break due to this

Reply:
I had my doubts about it as well.
`@frontify/sidebar-settings` is used in web-app (to be fixed in https://github.com/Frontify/web-app/pull/2899) and in themes (to be fixed in https://github.com/Frontify/guideline-themes/pull/59)

Reply:
Agreed. It is unfortunate, but this is a breaking change. This would best be done when we have some more breaking changes ready to be released. Until then, we will have to live with that typo